src/hotspot/share/cds/archiveHeapWriter.cpp

```diff
@@ -421,7 +366,7 @@ template <typename T> oop ArchiveHeapWriter::load_source_oop_from_buffer(T* buff
 
 template <typename T> void ArchiveHeapWriter::store_requested_oop_in_buffer(T* buffered_addr,
                                                                             oop request_oop) {
-  assert(is_in_requested_regions(request_oop), "must be");
+  //assert(is_in_requested_regions(request_oop), "must be");
```

Comment:
Some left over commented code. I assume this should be removed or a new assert should be here to replace it.